infra/src/dsl/bff.ts

```diff
@@ -29,17 +44,24 @@ export const bffConfig = ({
   clientName,
   clientId,
   globalPrefix,
-  allowedRedirectUris,
 }: BffInfo) => {
   const sanitizeGlobalPrefix = sanitizePath(globalPrefix)
 
-  const getBaseUrl = (ctx: Context) =>
-    ctx.featureDeploymentName
+  const getBaseUrl = (ctx: Context) => {
+    const domain = ctx.featureDeploymentName
       ? `${ctx.featureDeploymentName}-beta.${ctx.env.domain}`
       : ctx.env.type === 'prod'
       ? ctx.env.domain
```

Comment:
~Isn't `ctx.env.domain === 'island.is'` here? Could then get rid of `ISLAND_IS_PROD_URL` logic.